### DISCUSSION TITLE
Feature Request: Add an offset to logic of onMenuScrollToBottom

### DISCUSSION THREAD

**Discussion by omriavigdor** [JedWatson/react-select#5437] [Ideas]
I suggest exposing a prop to the user to set a pixel-offset for when onMenuScrollToBottom is called.
Meaning, suppose I want it to fire 20 pixels before the user scrolls to the bottom of the menu, then I set:
`menuScrollToBottomOffset={20}`

It will be very helpful if I can do my logic before the user gets to the bottom because if they do it fast enough the whole page is scrolled down (my case is implementing infinite scroll).

**Comment by brayhoward** [comment]
Would pay money for this feature, literally

**Comment by lukebennett88** [comment]
Thanks for opening this issue.
In an effort to stay on top of open issues, we're moving feature requests to discussions.
